### DISCUSSION TITLE
[closed] (old brief notes) Proximity

### DISCUSSION THREAD

**Discussion by shawna-slh** [w3c/low-vision-a11y-tf#108] [General]
Summary:
* User Need: Users know about and can find related information.
* [3.6.2 Proximity of Related Information](https://w3c.github.io/low-vision-a11y-tf/requirements.html#proximity-of-related-information) in User Needs / Requirements doc

Related guidance:
* [Ensure that form elements include clearly associated labels](https://www.w3.org/WAI/tips/designing/#ensure-that-form-elements-include-clearly-associated-labels) in Designing for Web Accessibility , Tips for Getting Started

Notes from elsewhere:
> This covers proximity of labels and controls — to make sure fields and labels are in same magnified viewport and in predictable locations such as checkbox to left of label — as well as proximity of on-screen keyboard to input fields - e.g. Desmos on iPad where these two things are not near each other.

**Comment by mraccess77** [comment]
In my opinion "User Need: Users know about and can find related information." does not describe the proximity issue.      It's perhaps something more like
* User's need the label and control or input field and on-screen keyboard to be in the same viewport at 400% magnification

That then leaves the related issue of feedback/changes that occur outside the viewport which may be more of an UAAG issue as there is not a good way to communicate live regions visually.

**Comment by shawna-slh** [comment]
closing as more is in https://github.com/w3c/low-vision-a11y-tf/discussions/133

**Comment by shawna-slh** [comment]
urrgg, seems there is no way to close or archive discussions. sigh. guess I'll just change the title :-/
